### PR TITLE
Codegen fix `fcvt_from_sint.f32 ` with small types on riscv64.

### LINKED ISSUE #5952: Cranelift: Wrong result for `fcvt_from_sint.f32` on RISCV
👋 Hey,

I re-ran the fuzzer today with the issues that were fixed in the past weeks, and it reported some new cases.

### `.clif` Test Case

```
test interpret
test run
target riscv64gc

function %a(i16 uext) -> f32 {
block0(v0: i16):
    v1 = fcvt_from_sint.f32 v0
    return v1
}

; run: %a(-12800) == -0x1.900000p13
```

### Steps to Reproduce

* `clif-util test ./the-above.clif`

### Expected Results

The test to pass!

### Actual Results

```
 ERROR cranelift_filetests::concurrent > FAIL: run
FAIL ./lmao.clif: run

Caused by:
    Failed test: run: %a(-12800) == -0x1.900000p13, actual: 0x1.9c0000p15
1 tests
Error: 1 failure
```

### Versions and Environment

Cranelift version or commit: main (e2dcb19099606f161c6d6145ba71130579b452e2)

Operating system: Linux

Architecture: RISCV64GC

### Extra Info

CC: @yuyang-ok

### PR DESCRIPTION
**Pull request by yuyang-ok**
This will fix #5952.

### CODE REVIEW

File: cranelift/codegen/src/isa/riscv64/inst.isle [hunk 1]
```diff
@@ -2122,6 +2122,13 @@
         (ones Reg (load_imm12 -1)))
   (value_reg (gen_select_reg (IntCC.Equal) zero input zero ones))))
 
+(decl normalize_fcvt_from_int (ValueRegs Type ExtendOp) ValueRegs)
```
Comment (afonso360) [inline cranelift/codegen/src/isa/riscv64/inst.isle:2125]: Can we move this next to the other `normalize` functions? Having it in the middle of the `lower_bmask` rules makes them somewhat difficult to read.

Comment (yuyang-ok) [reply cranelift/codegen/src/isa/riscv64/inst.isle:2125]: sure.

**Comment by yuyang-ok** [comment]
@afonso360 I have move the code.

**Review by afonso360 — APPROVED** [review]
LGTM! I suspect we're going to need something similar for the other `fcvt` ops, But let's see what the fuzzer has to say about it.

**Pull request merged**